### DISCUSSION TITLE
Discuss: projects vs single file editing

### DISCUSSION THREAD

**Discussion by VKondakoff** [zed-industries/zed#9853] [Feature Requests]
Hi!

Personally, I'm editing non-project files quite often. Unfortunately it looks like in the current state Zed lacks many features needed for single file editing (without loading any project).

For example:
Allow files to be added to Open recent together with folders (projects) https://github.com/zed-industries/zed/issues/9850
Restore non-project files after relaunching Zed https://github.com/zed-industries/zed/issues/9851
Add Find in open files functionality https://github.com/zed-industries/zed/issues/9852

Am I the only one here who noticed this? Any chance to change the situation?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
